vql/sigma/sigma_test.go

```diff
@@ -312,6 +312,40 @@ detection:
 					Set("Foo", "Bar"),
 			},
 		},
+		{
+			description: "Match single base64offset field",
+			rule: `
+title: Base64 offsets
+logsource:
+  product: windows
+  service: application
+
+detection:
+  selection1:
+     Foo|base64offset|contains: hello
+  selection2:
+     Foo|base64offset|contains: test
+  selection3:
+    Foo|base64offset|contains|all:
+      - velo
+      - ciraptor
+  selection4:
```

Comment:
This test seems to fail - should it match any of these terms?

Reply:
I have updated the tests 👍 Thanks for cleaning this up a bit.